server/sonar-server/src/main/java/org/sonar/server/ws/WebServiceEngine.java

```diff
@@ -112,9 +127,11 @@ public void execute(ValidatingRequest request, ServletResponse response,
     return action;
   }
 
-  private void sendErrors(ServletResponse response, int status, Errors errors) {
-    ServletResponse.ServletStream stream = response.stream();
-    stream.reset();
+  private void sendErrors(Response response, int status, Errors errors) {
+    Response.Stream stream = response.stream();
+    if (stream instanceof ServletResponse.ServletStream) {
```

Comment:
I find it suspicious that reset must be called here
can you explain why it is required?